mcp-spring/mcp-spring-webflux/src/main/java/io/modelcontextprotocol/server/transport/WebFluxStreamableServerTransportProvider.java

```diff
@@ -98,7 +98,7 @@ private WebFluxStreamableServerTransportProvider(ObjectMapper objectMapper, Stri
 
 	@Override
 	public List<String> protocolVersions() {
-		return List.of(ProtocolVersions.MCP_2024_11_05, ProtocolVersions.MCP_2025_03_26);
+		return List.of(ProtocolVersions.MCP_2025_03_26, ProtocolVersions.MCP_2025_06_18);
```

Comment:
Not sure if there can be more protocol, otherwise, we have to list all of them here, better to make it compareable.